src/plugins/enzymes/src/FindEnzymesTask.h

```diff
@@ -59,19 +59,34 @@ class FindEnzymesTask;
 
 struct FindEnzymesTaskConfig {
     FindEnzymesTaskConfig()
-        : maxResults(0x7FFFFFFF),
+        : maxResults(INT_MAX),
```

Comment:
Use `std::numeric_limits<int>::max()` instead of this ancient stuff

Reply:
I would like to keep standard INT_MAX for easier readability when possible.
limits<int>::max() return INT_MAX internally and both are a part of the standard.